src/game/etj_timerun_v2.cpp

```diff
@@ -913,6 +914,20 @@ bool ETJump::TimerunV2::isDebugging(int clientNum) {
   return false;
 }
 
+int ETJump::TimerunV2::indexForRunname(const std::string &runName) {
+  int index;
+  std::string currentRun;
+
+  for (index = 0; index < level.timerunNamesCount; index++) {
+    currentRun = sanitize(level.timerunNames[index], true);
+    if (currentRun == sanitize(runName, true)) {
```

Comment:
I'd probably extract the sanitize(runName, true) to a variable to reduce the unnecessary sanitization
I'd also consider a cache for these but I guess this isn't executed too often so prolly no point in that

Reply:
Done, ty.

Caching could maybe be done yea but we're probably talking about nanoseconds in execution time here so probably not critical, maybe later.